common/include/rbaudio.h

```diff
@@ -74,7 +75,41 @@ unsigned long RBAGetDiscID();
 
 // List the tracks on the CD
 void RBAList(void);
+
+#elif SDL_MAJOR_VERSION == 2
+
```

Comment:
Much of this appears to be duplicated from above.  Why not move the parts that are common out of the is-SDL-1 test, instead of duplicating common declarations in both blocks?